rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/HealthSpecTest.java

```diff
@@ -54,6 +54,8 @@ public void should_document_get_health() throws Exception {
                                 fieldWithPath("components.diskSpace.details.free").description("The free space of the diskspace."),
                                 fieldWithPath("components.diskSpace.details.threshold").description("The threshold of the diskspace."),
                                 fieldWithPath("components.diskSpace.details.exists").description("The existance of diskspace."),
+                                //TODO: Reverify
+                                fieldWithPath("components.diskSpace.details.path").description("The path of the file."),
```

Comment:
Should be where the fs is mounted?

Reply:
This field generally display the path being monitored for diskspace. For now its, where the application is build. The path can be configured in properties file.